gradle/libs.versions.toml

```diff
@@ -0,0 +1,23 @@
+[versions]
+gradle = "8.0.0"
+kotlin = "1.9.21"
+dokka = "1.9.10"
+ktlint = "11.3.1"
+detekt = "1.22.0"
+appcompat = "1.6.1"
+constraint-layout = "2.1.4"
```

Comment:
Can we just get rid of it? I don't think it's really needed for the sample.

Reply:
Agreed 👍